### DISCUSSION TITLE
How To Attach A File With Postal Mail Server WEB

### DISCUSSION THREAD

**Discussion by MSProgrammer** [postalserver/postal#1811] [Development discussions]
Hi Guys 

i want to Attach images And File with this Mail server

How It`s work with out any code And As default Postal Web server

thanks for Guiding us!

**Comment by MSProgrammer** [comment]
So there Is No Attachment Button in postal web Mailserver 

Please USE a Postal-API To send Any attachment you want

**Comment by willpower232** [comment]
Postal is designed to send email you give it so you need to generate a message using other software and pass it to Postal using API or SMTP. Both support attachments.

**Reply by MSProgrammer** [reply]
Thanks , Alot
